topo_processor/metadata/item.py

```diff
@@ -0,0 +1,29 @@
+from datetime import datetime
+
+import pystac as stac
+
+from .collection import Collection
+
+
+class Item:
+    path: str
+
+    def __init__(self, path: str, collection: Collection):
+        self.path = path
+        self.collection = collection
+        self.stac_item = stac.Item(
+            stac_extensions=[],
+            id="hash",  # TODO
+            geometry=None,  # TODO
+            bbox=None,  # TODO
+            properties={},
+            datetime=datetime.now(),
+        )
+
+        # # TODO: create thumbnail
```

Comment:
Creating thumbnails should not be done here, as a Item may not actually have a thumbnail to be created.